### DISCUSSION TITLE
Adding multiple line breaks between paragraphs

### DISCUSSION THREAD

**Discussion by ingridhoffman** [remarkjs/remark#887] [Q&A]
Hi. I am using react-markdown with remark-breaks to try to achieve multiple line breaks between my paragraphs.

With this test:
`const descriptionTest = 'line one \n line two \n\n line three \n\n\n line four';`

`<ReactMarkdown remarkPlugins={[remarkBreaks]} >{descriptionTest}</ReactMarkdown>`

I expect to see rendered:
line one\
line two\
\
line three\
\
\
line four    

And instead I see rendered:
  line one
  line two
  line three
  line four

Is it possible to get react-markup to respond to extra line breaks?

**Comment by wooorm** [comment]
The answer is: https://github.com/remarkjs/remark-breaks.

Please first search for your question through issues and discussions: https://github.com/remarkjs/react-markdown/issues?q=line+break, https://github.com/remarkjs/remark/discussions?discussions_q=line+break

**Reply by ingridhoffman** [reply]
Yes. As you might note in my code above, I am using remark-breaks. And I have read through all the discussions and tried the various hacks using non-breaking spacing, etc... but it seemed from the documentation that remark-breaks should be responding to multiple new lines in the way I was expecting. Am i misunderstanding the documentation or not using it correctly?

**Reply by wooorm** [reply]
oh you mean *between* paragraphs? No, that’s not what GitHub or `remark-breaks` does. HTML does not allow it. I also would not recommend it, because it’s not markdown.

CSS can. `white-space: pre` for example: https://developer.mozilla.org/en-US/docs/Web/CSS/white-space.

**Reply by ingridhoffman** [reply]
Thanks for the clarification. I'll play with using css and keep exploring.
UPDATE: css will not work for me here because react-markdown appears to strip out the newline characters when converting to html.
